File: go/iq/fb-thrift_iq~get~identifier.go
// BuildBizLinkingGetAccountNonce builds <iq type="get" xmlns="fb:thrift_iq">. Replies with one of: Success, Error.
func BuildBizLinkingGetAccountNonce(smaxId int, from types.JID, scope string) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"xmlns": "fb:thrift_iq", "smax_id": smaxId, "from": from, "to": "s.whatsapp.net", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "identifier", Attrs: waBinary.Attrs{"scope": scope}}, // optional
		},
	}
}

File: rust/iq/fb-thrift_iq~get~identifier.rs
/// <iq type="get" xmlns="fb:thrift_iq">. Replies with one of: Success, Error.
pub fn build_biz_linking_get_account_nonce(smaxId: i64, from: Jid, scope: String) -> Node {
    NodeBuilder::new("iq")
        .attr("xmlns", "fb:thrift_iq")
        .attr("smax_id", smaxId)
        .attr("from", from)
        .attr("to", "s.whatsapp.net")
        .attr("type", "get")
        .child(NodeBuilder::new("identifier").attr("scope", scope).build()) // optional
        .build()
}